Library/Formula/camlp5.rb

```diff
@@ -11,12 +11,13 @@ class Camlp5 < Formula
     sha256 "60312c75054d1db597c8afaf2d0563d41e192480531b74b2ad40908e36bbfb8b" => :mountain_lion
   end
 
-  depends_on "ocaml"
+  deprecated_option "strict" => "with-strict"
+  option "with-strict", "Compile in strict mode"
```

Comment:
Added a fix. It now recognizes the `--with-strict`.

```
$ brew install camlp5 --with-strict
[...]
🍺  /usr/local/Cellar/camlp5/6.14: 204 files, 26M, built in 43 seconds
[/usr/local/Library/Formula on ⇄ audit-fix-C-01]
$ brew install coq
coq: camlp5 must be compiled in transitional mode (instead of --strict mode):
  brew install camlp5
Error: An unsatisfied requirement failed this build.
[✘ /usr/local/Library/Formula on ⇄ audit-fix-C-01]
$
```